helm/cluster-operator/templates/rbac.yaml

```diff
@@ -51,14 +51,23 @@ rules:
     verbs:
       - "*"
   - apiGroups:
-      - cluster.k8s.io
+      - cluster.x-k8s.io
```

Comment:
What does `cluster.x-k8s.io`  mean?

Reply:
That is the fancy shit from upstream: https://github.com/kubernetes-sigs/cluster-api/blob/master/config/crd/bases/cluster.x-k8s.io_clusters.yaml.